src/CheckerErrors.js

```diff
@@ -0,0 +1,30 @@
+function checkerErrors(messageToConvert, paramsToConvert) {
+  const convertParameters = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+  const errors = [];
+
+  if (!convertParameters.includes(paramsToConvert)
+      && paramsToConvert !== null) {
+    errors.push({
+      // eslint-disable-next-line max-len
+      message: 'This case is not supported. Available cases: SNAKE, KEBAB, CAMEL, PASCAL, UPPER.',
+    });
+  }
+
+  if (!messageToConvert) {
+    errors.push({
+      // eslint-disable-next-line max-len
+      message: 'Text to convert is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
```

Comment:
Save your correct format (`"/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>"`) to a const variable, and put it in message using a [template string literal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals). 

That way, you will not repeat it, and you will not need to make eslint disable comments.
For the `This case is not supported...` message, you can put `convertParameters.join(', ')` inside a template string.